number.py

```diff
@@ -1,28 +1,4 @@
 # Import the math module using import keyword.
 import math
-# Give the list as static input and store it in a variable.
-list1 = [4, 1, 4, 145]
```

Comment:
This is a Unusedimportsys
```suggestion 
```

 from our GitHub Enhacement Bot